src/components/Messages/Message/index.js

```diff
@@ -88,7 +88,8 @@ export const Message = memo(({
 	>
 		<MessageAvatars
 			avatarResolver={avatarResolver}
-			usernames={compact ? [] : message.u && message.u.username && [message.u.username]}
+			// eslint-disable-next-line no-nested-ternary
+			usernames={compact ? [] : message.alias ? message.u && message.u.name && [message.u.name] : message.u && message.u.username && [message.u.username]}
```

Comment:
I think extending ternary operators is making hard to read the logic. My suggestion is to create a helper method to deal and return the expected `usernames`.

```suggestion
			usernames={this.getMessageUsernames(message)}
```